Author: Andreas Hoenen <andreas@hoenen-terstappen.de>
Description: Hotfix for BTS report #576217:
Avoid pdflatex runs repeating ad infinitum in case of:
- <book> document with german umlauts in headings
- dblatex processing options:
  * latex.encoding=utf8
  * latex.unicode.use=1
Please note that this patch only resolves the special case reported to BTS,
a fundamental solution must take into account also other non-ASCII characters,
e.g. î, …
--- a/xsl/lang.xsl
+++ b/xsl/lang.xsl
@@ -252,6 +252,7 @@
     <xsl:text>\usepackage[T2A,T2D,T1]{fontenc}&#10;</xsl:text>
     <xsl:text>\usepackage{ucs}&#10;</xsl:text>
     <xsl:text>\usepackage[utf8x]{inputenc}&#10;</xsl:text>
+    <xsl:text>\PrerenderUnicode{äöüÄÜÖß}&#10;</xsl:text>
     <xsl:text>\def\hyperparamadd{unicode=true}&#10;</xsl:text>
   </xsl:when>
   </xsl:choose>
